ovsdb/client.go

```diff
@@ -229,6 +262,25 @@ func (c *Client) listen() {
 	}
 }
 
+// echoLoop starts a loop that sends echo RPCs at the interval defined by d.
+func (c *Client) echoLoop(ctx context.Context, d time.Duration) {
+	t := time.NewTicker(d)
+	defer t.Stop()
+
+	for {
+		select {
+		case <-ctx.Done():
+			return
+		case <-t.C:
+		}
+
+		// No feasible way to handle errors here.  In the future, it may be
+		// possible to do something like re-establishing the connection.
+		// TOOD(mdlayher): improve error handling for echo loop.
+		_ = c.Echo(ctx)
```

Comment:
yep that's what I was gonna ask! I have used something like an error channel and then a function in main that just reads from the error channel and logs to stdout with some key-value pairs. Unfortunately my particular example involved shoving an error channel into ctx but maybe there's a better way around this?

Reply:
I think that there are a couple of options here:

- increment some counters for the stats type with success/failure of echo
- log on error using a user-specified logger
- attempt to reconnect the connection

I want to try the third one and make this client smarter, but I don't think it's feasible in this PR.

I could do one or both of the first two options though. What do you think?

Reply:
I suppose  it depends on what you hope to achieve by documenting the errors - i.e. just inform users they are happening or actually introspect into the error itself?

Reply:
Yep, let's stick with information for now. I will add a commit that adds counters here.

Reply:
great!